codalab/apps/web/models.py

```diff
@@ -1801,6 +1830,35 @@ def dict_constructor(loader, node):
                         assert False, "OrganizerDataSet (%s) could not be found" % phase_spec["starting_kit"]
                         # End unpack starting kit
 
+            # Begin unpack public data
+            if hasattr(phase, 'public_data') and phase.public_data:
+                if phase_spec["public_data"].endswith(".zip"):
```

Comment:
this doesn't handle multiple `public_datas` I think?

Reply:
No, is that something we want? It might be a little redundant with one public data field per Phase, and a starting kit field per phase.

Reply:
Yep yep, if we're replacing that old `data` stuff, that handled multiple External Files or whatever right? I sent you some DMs about a plan to add a new model to point many `PublicDatas` at 1 competition

Reply:
Isabelle decided against this, since we can group together organizer datasets.